Resources/templates/responsive/channel/call/partials/banner_header.php

```diff
@@ -24,14 +25,14 @@
 			<div class="row">
 				<div class="col-lg-6 col-md-7 col-sm-8 col-xs-12">
 					<div>
-						<span class="title" style="<?= $this->colors['header'] ? "color:".$this->colors['header'].";" : '' ?> <?= $this->colors['header_shadow'] ? "text-shadow:none" : '' ?>" >
-						<?= $this->channel->subtitle ?>
+						<span class="title" style="<?= $colors['header'] ? "color:".$colors['header'].";" : '' ?> <?= $colors['header_shadow'] ? "text-shadow:none" : '' ?>" >
```

Comment:
Can we use single quotes (') instead of double quotes for the HTML values in here? E.g `<?= $colors['header'] ? 'color:'.$color['header'].';'`.

It would help readability and very slightly improve performance since PHP doesn't need to evaluate those strings.